android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMHelpFileActivity.java

```diff
@@ -42,13 +44,27 @@
   private WebView webView;
   private Button finishButton;
   private String pkgID;
+  private static boolean didSetDataDirectorySuffix = false;
 
   @SuppressLint({"SetJavaScriptEnabled", "InflateParams"})
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     final Context context = this;
 
+    // Teardown keyboards to free up webView
+    KMManager.onDestroy();
```

Comment:
I think we should probably rename the function and call it from `onDestroy` -- `onDestroy` should be reserved for the event handler.

Reply:
After some thought, I'm even more concerned about this pattern. Should we be setting `KMManager` to `null` after this? What state is it in after calling `onDestroy`? Do we have any API contract around this?

Reply:
Turns out this wouldn't help with rendering sil_cameroon_qwerty. So I've opted not to keep this change